source/common/network/utility.cc

```diff
@@ -133,14 +141,18 @@ Address::InstanceConstSharedPtr Utility::parseInternetAddressNoThrow(const std::
   if (inet_pton(AF_INET, ip_address.c_str(), &sa4.sin_addr) == 1) {
     sa4.sin_family = AF_INET;
     sa4.sin_port = htons(port);
-    return std::make_shared<Address::Ipv4Instance>(&sa4);
+    StatusOr<Address::InstanceConstSharedPtr> address =
+        Address::InstanceFactory::createInstancePtr<Address::Ipv4Instance>(&sa4);
+    return instanceOrNull(address);
```

Comment:
You can also avoid writing the type here by doing:
```
  return addressOrNull(InstanceFactory::createInstancePtr<Address::Ipv4Instance>(&sa4));
```
I'm assuming here (as above) that the `Address::` qualification is not needed.

Here and below.

Reply:
This is in namespace Envoy::Network::. so we need Address:: if we are using class and method in  Envoy::Network::Address::